src/client/settings/Settings.js

```diff
@@ -355,6 +366,26 @@ export default class Settings extends React.Component {
                     </Checkbox>
                   </div>
                 </div>
+                <div className="Settings__section">
+                  <h3>
+                    <FormattedMessage id="upvote_setting" defaultMessage="Automatically upvote" />
+                  </h3>
+                  <p>
+                    <FormattedMessage
+                      id="upvote_setting_details"
+                      defaultMessage="Enable this option to automatically upvote your own posts."
```

Comment:
This needs to be in `default.json` also as well as any new text. Check with @bonustrack for for text approval.

Reply:
We should use `like` instead of `upvote`, similar than the others place in the app. Also i would change the title to `Like my posts`. `Automatically upvote` feel confusing, this title doesn't say that it's only in your own post.